### PR TITLE
test(e2e) MQ-2644 fix pvc deletion

### PR DESCRIPTION
**Pull request by milanhajek**

### CODE REVIEW

**Review by chriswldenyer — COMMENTED** [review]
Two things to check.

File: src/MQ-2644-invalid_volume_sizes/invalid_volume_sizes_test.go [hunk 1]
```diff
@@ -41,38 +41,33 @@ var _ = Describe("Test invalid volume sizes", func() {
 	})
 
 	It("should verify to not create pvc with negative size", func() {
-		testNames = nil
 		c := generatePvc("negative-size", 3, -1000)
 		c.pvcZeroOrNegativeSizeTest()
-		cleanUp()
+		deleteSC(c.testName)
```
Comment (chriswldenyer) [inline src/MQ-2644-invalid_volume_sizes/invalid_volume_sizes_test.go:46]: should this be deleteSc(c.scName) ?

File: src/MQ-2644-invalid_volume_sizes/invalid_volume_sizes_test.go [hunk 2]
```diff
@@ -41,38 +41,33 @@ var _ = Describe("Test invalid volume sizes", func() {
...
 	})
 
 	It("should verify to not create pvc with zero size", func() {
-		testNames = nil
 		c := generatePvc("zero-size", 3, 0)
 		c.pvcZeroOrNegativeSizeTest()
-		cleanUp()
+		deleteSC(c.testName)
```
Comment (chriswldenyer) [inline src/MQ-2644-invalid_volume_sizes/invalid_volume_sizes_test.go:52]: also here should this be deleteSc(c.scName) ?

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Review by blaisedias — APPROVED** [review]
LGTM

**Comment by milanhajek** [comment]
bors merge

**Pull request merged**